Makefile

```diff
@@ -115,6 +125,9 @@ clean-ebpf:
 
 clean: clean-ebpf
 	$(call out_console,CLEAN,cleaning retis ...)
+ifeq ($(COV),1)
+	$(CARGO) llvm-cov clean --workspace
```

Comment:
given `./target` gets deleted here, this call is not strictly required.
It would be nice instead having a `clean-cov` target with the same command (added to `.PHONY` as well)